extensions/interactions/base_test.py

```diff
@@ -246,6 +248,73 @@ def test_interaction_rule_descriptions_in_dict(self):
             'IsWithinTolerance': 'is within {{tol|Real}} of {{x|Real}}'
         })
 
+    def test_html_field_types_to_rule_specs_mapping_are_valid(self):
```

Comment:
Btw this whole method is way cleaner now than the first draft. Thanks for tidying it up!